docs/modules/serialization/pages/serialization.adoc

```diff
@@ -21,7 +21,8 @@ Hazelcast offers the following serialization options:
 | Serialization interface| Advantages| Disadvantages|Client support
 
 | xref:compact-serialization.adoc[Compact Serialization]
-| * More memory usage efficient than `Portable`
+Recommended
```

Comment:
Maybe needs either some formatting or expanded into a sentence e.g. Recommended for...